src/app/action-items/action-items.component.spec.ts

```diff
@@ -35,42 +27,38 @@ describe('ActionItemsComponent', () => {
     expect(component).toBeTruthy();
   });
 
-  it('should stop loading and render the view after 1s', done => {
-    component.loading = true;
+  it('should return an array with 1 object', async(() => {
```

Comment:
This test states that it validates if there's a single object but it's testing on empty array and not checking the 1 object but the type of array.